Makefile

```diff
@@ -30,28 +30,28 @@ prebuild:
 	mkdir -p  $(BUILD_DIR)
 
 api: prebuild
-	CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s -extldflags "-static"' -o $(BUILD_DIR)/api github.com/opensds/multi-cloud/api/cmd
+	CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s -extldflags "-static"' -o $(BUILD_DIR)/api github.com/soda/multi-cloud/api/cmd
```

Comment:
Why we are not making full changes as per current repo and org name.
Replace `opensds/multi-cloud`  with  `sodafoundation/strato`